docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -10,10 +10,6 @@ A `dense_vector` field stores dense vectors of float values.
 The maximum number of dimensions that can be in a vector should
 not exceed 2048. A `dense_vector` field is a single-valued field.
 
-These vectors can be used for <<vector-functions,document scoring>>.
```

Comment:
I'm curious why we removed this paragraph?

Reply:
It was not removed, just moved down. I put it back to its original place in 34d961b92339750bdd649bcb5e0b2064609fc578.

Reply:
Sorry, I read this part too quickly.